rgd/geodata/management/commands/gc_landsat.py

```diff
@@ -0,0 +1,145 @@
+from datetime import datetime
+
+import pandas as pd
+
+from rgd.geodata import datastore
+
+from . import _data_helper as helper
+
+SUCCESS_MSG = 'Finished loading all landsat data.'
+
+
+def _fetch_index_table():
+    # https://data.kitware.com/#item/605cbf9c2fa25629b9e15c5c
+    path = datastore.datastore.fetch('landsat_korea.csv')
+    return pd.read_csv(path)
+
+
+def _format_base_url(base_url):
+    return 'http://storage.googleapis.com/' + base_url.replace('gs://', '')
+
+
+def _make_image_urls(base_url, name, sensor='TM'):
+    if sensor == 'TM':
+        possible_bands = [
+            'B1.TIF',
+            'B2.TIF',
+            'B3.TIF',
+            'B4.TIF',
+            'B5.TIF',
+            'B6.TIF',
+            'B7.TIF',
+            'GCP.txt',
+            'VER.txt',
+            'VER.jpg',
+            'ANG.txt',
+            'BQA.TIF',
+        ]
+        possible_anc = [
+            'ANG.txt',
+            'MTL.txt',
+        ]
+    elif sensor in [
+        'OLI_TIRS',
+        'OLI',
+    ]:  # Landsat 8
+        possible_bands = [
+            'B1.TIF',
+            'B2.TIF',
+            'B3.TIF',
+            'B4.TIF',
+            'B5.TIF',
+            'B6.TIF',
+            'B7.TIF',
+            'B8.TIF',
+            'B9.TIF',
+            'B10.TIF',
+            'B11.TIF',
+            'BQA.TIF',
+        ]
+        possible_anc = [
+            'MTL.txt',
+        ]
+    elif sensor == 'ETM':  # Landsat 7
+        possible_bands = [
+            'B1.TIF',
+            'B2.TIF',
+            'B3.TIF',
+            'B4.TIF',
+            'B5.TIF',
+            # 'B6.TIF',
+            'B6_VCID_1.TIF',
+            'B6_VCID_2.TIF',
+            'B7.TIF',
+            'B8.TIF',
+            # 'B9.TIF',
+            'BQA.TIF',
+        ]
+        possible_anc = [
+            'MTL.txt',
+        ]
+    else:
+        possible_bands = [
+            'B1.TIF',
+            'B2.TIF',
+            'B3.TIF',
+            'B4.TIF',
+            'B5.TIF',
+            # 'B6.TIF',
+            'B6_VCID_1.TIF',
+            'B6_VCID_2.TIF',
+            'B7.TIF',
+            'B8.TIF',
+            'B9.TIF',
+            'BQA.TIF',
+        ]
+        possible_anc = [
+            'MTL.txt',
+        ]
```

Comment:
Needs double checking

Reply:
@matt-bernstein, could you do a sanity check on this?

Reply:
@banesullivan
This ingestion shouldn't touch `sensor == 'TM'`, as that's from Landsat 4-5, which were discontinued in 2013. You probably also want to move the non-`TIF` files from `possible_bands` to `possible_anc`.

For Landsat 7 and 8, there is another ancillary file `ANG.txt`. (ex. [L7](https://console.cloud.google.com/storage/browser/gcp-public-data-landsat/LE07/01/001/001/LE07_L1GT_001001_20010715_20170204_01_T2), [L8](https://console.cloud.google.com/storage/browser/gcp-public-data-landsat/LC08/01/001/002/LC08_L1GT_001002_20160817_20170322_01_T2)) I am not sure why this file was missing in the scenes I downloaded using [FeLS](https://gitlab.kitware.com/smart/fetchLandsatSentinelFromGoogleCloud) and [Landsat578](https://gitlab.kitware.com/smart/Landsat578). Maybe it only shows up sometimes. It does look like a valid angle map to me.

For Landsat 8, this is otherwise correct for `sensor == 'OLI_TIRS'`

The sensors `OLI` and `TIRS` alone ([ex](https://console.cloud.google.com/storage/browser/gcp-public-data-landsat/LO08/01/002/010/LO08_L1GT_002010_20150211_20170413_01_T2), [ex](https://console.cloud.google.com/storage/browser/gcp-public-data-landsat/LT08/01/001/248/LT08_L1GT_001248_20130318_20170505_01_T2)) contain only B1-9 and B10-11 respectively. These are the visual vs. thermal bands. 

These 3 sensor types for Landsat 8 map to `LC08/`, `LO08/`, and `LT08/` respectively - see [here](https://gitlab.kitware.com/smart/Landsat578#-1) for nomenclature.

For now I would not download `OLI` or `TIRS` alone. There is probably a good reason the universities and FeLS/Landsat578 ignore them, though I will follow up next week to confirm.

Reply:
Thanks for the review!